spring-cloud-huawei-service-engine/service-engine-common/src/main/java/com/huaweicloud/service/engine/common/configration/bootstrap/DiscoveryBootstrapProperties.java

```diff
@@ -74,6 +74,8 @@ public class DiscoveryBootstrapProperties {
   //          enabled: true
   private boolean enableServicePolling = false;
 
```

Comment:
It's better to use a `boolean` configuration to check if prefer ipv4 or ipv6.

Reply:
fixed